### PR TITLE
[docs] [2.20.4] Added more alert templates

### PR DESCRIPTION
**Pull request by aishwarya24**
https://deploy-preview-22116--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 1]
```diff
@@ -267,6 +282,28 @@ Master is missing from Raft group or has follower lag higher than `$threshold` s
...
+```expression
+(avg by (universe_uuid) (rate(rpc_latency_sum{export_type="tserver_export", service_type="TabletServerService", universe_uuid="__universeUuid__", server_type="yb_tserver", service_method="Read"}[5m])))
+       / (avg by (universe_uuid)
+(rate(rpc_latency_count{export_type="tserver_export", service_type="TabletServerService", universe_uuid="__universeUuid__", server_type="yb_tserver", service_method="Read"}[5m])) * 1000)
+      {{ query_condition }} {{ query_threshold }}
+```
+
+#### Tablet server average write latency is high
+
+Average write latency of tablet server for universe '{{ $labels.source_name }}' `'$universe_name'` is above `$threshold%` ms. The current value is `$value` milliseconds.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:298]: labels.source_name  - backticks? and this variable isn't in read latency above - is that right?

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 2]
```diff
@@ -339,6 +376,47 @@ More recent OS version is recommended for this universe. Consider running VM ima
...
+
+RPC queue size is high for universe `'$universe_name'`.
+
+```expression
+max by (universe_uuid) (min_over_time({universe_uuid="__universeUuid__", saved_name=~"rpcs_in_queue_.*", export_type="tserver_export"}[5m]) {{ query_condition }} {{ query_threshold }})
+```
+
+#### Log cache size is high
+
+WAL cache size is high for nodes `'$node_name'` in universe `'$universe_name'`. The current value is `$value` MB for one of the nodes.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:414]: If it is WAL, maybe heading should be as well?

**Review by ddhodge — APPROVED** [review]
misc comments

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md (hunk 1 above)
Comment (Sahith02) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:298]: Yeah I think the variable `labels.source_name` is not supposed to be here?

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 3]
```diff
@@ -174,6 +174,21 @@ Test YSQL write/read operation failed on `$value` nodes for universe `'$universe
 count by (node_prefix) (yb_node_ysql_write_read{node_prefix="$node_prefix"} < 1)
 ```
 
+#### DocDB cache miss percentage is high
+
+DocDB cache miss percentage is high for universe `'$universe_name'`. The current value is `$value` %.
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:179]: @SravanTailam Could you please confirm if this info is okay for all these new alerts or do you want to add defaults as well?

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 4]
```diff
@@ -174,6 +174,21 @@ Test YSQL write/read operation failed on `$value` nodes for universe `'$universe
...
+
+```expression
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:181]: change all to promql

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 5]
```diff
@@ -131,39 +131,77 @@ sum by (universe_uuid) (ybp_health_check_node_master_fatal_logs{universe_uuid="$
...
+  1
 ```
 
 #### DB memory overload
 
 Database memory rejections have been detected for universe `'$universe_name'`.
 
 ```promql
-sum by (node_prefix) (increase(leader_memory_pressure_rejections{node_prefix="$node_prefix"}[10m])) + sum by (node_prefix) (increase(follower_memory_pressure_rejections{node_prefix="$node_prefix"}[10m])) + sum by (node_prefix) (increase(operation_memory_pressure_rejections{node_prefix="$node_prefix"}[10m])) > 0
+      sum by (node_prefix) (increase(leader_memory_pressure_rejections{node_prefix="$node_prefix"}[10m]))
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:182]: is this indentation correct?

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 6]
```diff
@@ -131,39 +131,77 @@ sum by (universe_uuid) (ybp_health_check_node_master_fatal_logs{universe_uuid="$
...
+  0
 ```
 
 #### DB version mismatch
 
 Version mismatch has been detected for universe `'$universe_name'` for `$value` Master or T-Server instances.
 
 ```promql
-ybp_health_check_tserver_version_mismatch{universe_uuid="$uuid"} + ybp_health_check_master_version_mismatch{universe_uuid="$uuid"} > 0
+   ybp_health_check_tserver_version_mismatch{universe_uuid="$uuid"}
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:200]: also looks a bit off

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md (hunk 5 above)
Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:182]: Yes this is from the PromQL parser - https://o11y.tools/promqlparser/

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md (hunk 6 above)
Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:200]: same with this. Verified it again with the parser.

**Pull request merged**